containerfiles/Dockerfile

```diff
@@ -90,13 +66,11 @@ RUN \
       TARGET_TRIPLE=$(uname -m)-unknown-linux-gnu; \
     fi; \
     export PROTOC=/usr/local/bin/protoc; \
-    cargo build --release --target $TARGET_TRIPLE --bin $TARGETBINARY;
-
-# replace this with `--out` or `--out-dir` once stable
-RUN mkdir -p target/release
-RUN if [ "$TARGETPLATFORM" = "linux/arm64" ]; then ARCH=aarch64; \
+    cargo build --release --target $TARGET_TRIPLE --bin $TARGETBINARY;\
+    # Copies the binary from out of the cache directory
+    if [ "$TARGETPLATFORM" = "linux/arm64" ]; then ARCH=aarch64; \
     elif [ "$TARGETPLATFORM" = "linux/amd64" ]; then ARCH=x86_64; fi; \
-    cp target/$ARCH-unknown-linux-gnu/release/$TARGETBINARY target/release/
+    cp target/$ARCH-unknown-linux-gnu/release/$TARGETBINARY release/;
```

Comment:
this used to be a seperate step, but now that it is in the cache, made a single step. Copies the executable out of the cache directory into a seperate one. We might be able to replace with `out_dir` if/when that is stabilized.